test/clientTest.js

```diff
@@ -25,11 +24,9 @@ mc.supportedVersions.forEach(function (supportedVersion, i) {
     minMem: 1024,
     maxMem: 1024
   })
-  wrap.on('line', function (line) {
-    console.log(line)
-  })
+  // wrap.on('line', console.log);
 
-  describe('client ' + version.minecraftVersion, function () {
+  describe(`Testing minecraft client ${version.minecraftVersion}`, function () {
```

Comment:
Should be "Testing minecraft-protocol client" (this is using the minecraft server)

Reply:
Will update with more precise texts